fix(sessions): :pencil2: edits to basics session (#151)
rostools/github-intro: sessions/basics.qmd

it does, using the table below as a guide.

Also briefly introduce Markdown and mention the `.md` file extension.

> You can verbally relate `.md` to `.qmd`, but not get into it too much.

Reviewer: ```suggestion
You can verbally relate `.md` to `.qmd`, but not get into the details of it too much.
```

::: {.callout-note collapse="true"}
## :teacher: Teacher note

> Reinforce that paths are pointers to files on your computer, that they
> are for us humans to effectively organise and work with files. That
> every file has a parent folder, and every folder may also have a parent
> folder. That files and folders are separated by `/` or `\` and that the
> last item in the path is either a file or a folder. That a path is like
> a family tree, where each parent can have one or more child, but each
> child only has two sets of parents.

Reviewer: ```suggestion
Reinforce that: 

- Paths are pointers to files on your computer
- They are for us humans to effectively organise and work with files
-  Every file has a parent folder, and every folder may also have a parent
folder
- Files and folders are separated by `/` or `\` and that the
last item in the path is either a file or a folder. 
- A path is like a family tree, where each parent can have one or more child, but each
child only has two sets of parents.
```
Not sure I understand the last point? Can you elaborate/rewrite?

Author: I just removed it :relaxed:

    └── baked-goods/cake.md
    ```

Because the Git history consists of these connected commits that each

> have the changes, we're able to see exactly what was changed. You can

Reviewer: ```suggestion
contain changes, we're able to see exactly what was changed. You can
```

> even go back to the state of the folder, with the exact files and their
> contents that existed at that point in time.

Reviewer: ```suggestion
even go back to the state of the folder or file and see
the exact contents that existed at that point in time.
```
